### PR TITLE
update(doc) for Lenovo XCC SNMP

### PR DESCRIPTION
**Pull request by ponchoh**
## Description

- use template
- update content
- add links

REFS: MON-16610

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] Cloud (staging)
- [x] Plugin Packs (staging)
- [ ] 22.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md [hunk 1]
```diff
@@ -2,34 +2,139 @@
...
+| Alias           | Modèle de service                  | Description                    | Défaut |
+|:----------------|:-----------------------------------|:-------------------------------|:-------|
+| Hardware-Global | HW-Lenovo-Xcc-Hardware-Global-SNMP | Contrôle l'ensemble des sondes | X      |
+
+### Métriques & statuts collectés
+
+<Tabs groupId="sync">
+<TabItem value="Hardware-Global" label="Hardware-Global">
+
+Could not retrive metrics
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md:28]: You have to manually add metrics when script cannot figure out which ones it should add

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md [hunk 2]
```diff
@@ -2,34 +2,139 @@
...
+## Configuration
+
+### hôte
+
+* Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
+* Complétez les champs **Nom**, **Alias** & **IP Address/DNS** correspondant à votre serveur **Lenovo XCC SNMP**.
+* Appliquez le modèle d'hôte **HW-Server-Lenovo-Xcc-SNMP-custom**.
+
+Si vous utilisez SNMP en version 3, vous devez configurer les paramètres
+spécifiques associés via la macro **SNMPEXTRAOPTIONS**.
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md:88]: Should be updated according to latest layout. Ref https://github.com/centreon/centreon-documentation/pull/1716/files

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md [hunk 3]
```diff
@@ -2,34 +2,139 @@
...
+    --component='.*' \
+    --verbose \
+    --use-new-perfdata
+```
+
+La commande devrait retourner un message de sortie similaire à :
 
-SNMP must be configured on the monitored host
+```bash
+OK: |
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md:115]: Output is probably not just OK

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md [hunk 4]
```diff
@@ -2,34 +2,139 @@
...
-| :----------------------------------- | :------------------------------- |
-| Host name                            | *Name of the host*               |
-| Alias                                | *Host description*               |
-| IP                                   | *Host IP Address*                |
-| Monitored from                       | *Monitoring Poller to use*       |
-| Host Multiple Templates              | HW-Server-Lenovo-Xcc-SNMP-custom |
+### Diagnostic des erreurs communes
 
-Click on the *Save* button.
+Rendez-vous sur la [documentation dédiée](../getting-started/how-to-guides/troubleshooting-plugins.md)
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md:139]: Update link to the specific snmp troubleshooting section

File: pp/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md [hunk 5]
```diff
@@ -2,34 +2,140 @@
 id: hardware-servers-lenovo-xcc-snmp
 title: Lenovo XCC SNMP
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md:6]: Check my comments on the FR version.

**Review by Sims24 — CHANGES_REQUESTED** [review]
Few things to modify as I took so much time to review this PR, it results on some extra work on our side. 

Thanks who? 🙃

**Comment by omercier** [comment]
A Jira ticket has been created
REFS: MON-16610

**Review by omercier — APPROVED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md [hunk 6]
```diff
@@ -2,34 +2,192 @@
...
 title: Lenovo XCC SNMP
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
+## Vue d'ensemble
 
-### Centreon Plugin
+Lenovo développe, fabrique et vend du matériel et logiciels informatiques.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md:10]:
```suggestion
Lenovo développe, fabrique et vend du matériel et des logiciels informatiques.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md [hunk 7]
```diff
@@ -2,34 +2,192 @@
...
+
+## Installation
+
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet
+n'est pas requise pour voir apparaître le pack dans le menu **Configuration > Plugin Packs > Gestionnaire**.
+
+Au contraire, si la plateforme utilise une licence *offline*, installez le paquet
+sur le **serveur central** via la commande correspondant au gestionnaire de paquet
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md:67]:
```suggestion
sur le **serveur central** via la commande correspondant au gestionnaire de paquets
```

File: pp/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md [hunk 8]
```diff
@@ -2,34 +2,196 @@
...
+
+
 
 ## Prerequisites
 
-### Centreon Plugin
+### SNMP Configuration
+
+To use this pack, the SNMP service must be properly configured on your **Lenovo XCC SNMP**
+server. Please refer to the official documentation from XXX:
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md:53]:
```suggestion
server. Please refer to the official documentation from Lenovo:
```

File: pp/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md [hunk 9]
```diff
@@ -2,34 +2,196 @@
...
+### Host
 
-### Create a new host
+* Log into Centreon and add a new host through **Configuration > Hosts**.
+* Fill the **Name**, **Alias** & **IP Address/DNS** fields according to your **Lenovo XCC SNMP** server settings.
+* Apply the **HW-Server-Lenovo-Xcc-SNMP-custom** template to the host.
 
-Go to *Configuration \> Hosts* and click *Add*. Then, fill the form as shown by
-the following table:
+If you are using SNMP Version 3, use the **SNMPEXTRAOPTIONS** macro to configure
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-servers-lenovo-xcc-snmp.md:143]:
```suggestion
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**